src/index.js

```diff
@@ -38,7 +39,7 @@ export function pitch(request) {
 
   const filename = loaderUtils.interpolateName(
     this,
-    options.name || '[hash].worker.js',
+    options.name || this._compilation.outputOptions.filename,
```

Comment:
By default this should from https://webpack.js.org/configuration/output/#outputfilename